### PR TITLE
Update animals.yml

### PR DESCRIPTION
**Pull request by Sinis-Boop**
Update Animals.yml design doc:

Added component "CanEnterPipeCrawling" Which, in this case, gives the mob in question (id: MobDianoNymph) the ability to crawl in vents and pipes.

Why:
Diano Nymphs are small, just like mice, geras, and mothroaches, so I think nymphs should be able to vent crawl too.

Balance: Diona Nymphs are incapable of doing anything on their own as a Nymph, which makes it very balanced for them to be able to move (pretty much) everywhere across the station. Turning into a nymph comes at a great cost of losing everything, including identity, so I think it's a fair trade off.

Gwen did NOT kidnap me and force me to change this line of code and i am not in need of any immediate rescue.

:3

### CODE REVIEW

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 1]
```diff
@@ -3746,8 +3746,9 @@
   id: MobDionaNymph
   description: It's like a cat, only.... branch-ier.
   components:
+  - type: CanEnterPipeCrawling
   - type: Sprite
-    drawdepth: Mobs
+  drawdepth: Mobs
```
Comment (Doctor-Cpu) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml:3751]: indentation
needs one more tab so its under sprite

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 2]
```diff
@@ -3746,8 +3746,9 @@
   id: MobDionaNymph
   description: It's like a cat, only.... branch-ier.
   components:
+  - type: CanEnterPipeCrawling
```
Comment (Doctor-Cpu) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml:3749]: move this down to like the bottom of this idk

**Review by Doctor-Cpu — CHANGES_REQUESTED** [review]
:trollface:

**Comment by Sinis-Boop** [comment]
> :trollface:
:trollface:

**Pull request closed without merging**